iframes/identity/userCountry.js

```diff
@@ -0,0 +1,28 @@
+export const fetchUserCountry = () => {
+  const US = 'US';
+  const geoLocationUrl = 'https://www.paypalobjects.com/muse/noop.js';
+
+  return fetch(geoLocationUrl, { mode: 'no-cors' })
```

Comment:
is fetch a country call goes via iframe? if so do we still need no-cors parameter

Reply:
Yes we do. Iframe and noop .js live on different domains. `paypal.com` vs `paypalobjects.com`